src/sentry/api/endpoints/organization_profiling_profiles.py

```diff
@@ -41,3 +42,20 @@ def get(self, request: Request, organization: Organization) -> HttpResponse:
         kwargs = {"params": params}
 
         return proxy_profiling_service("GET", f"/organizations/{organization.id}/filters", **kwargs)
+
+
+@region_silo_endpoint
+class OrganizationProfilingFlamegraphEndpoint(OrganizationProfilingBaseEndpoint):
+    def get(self, request: Request, organization: Organization) -> HttpResponse:
+        if not features.has("organizations:profiling", organization, actor=request.user):
+            return Response(status=404)
+
+        params = self.get_snuba_params(request, organization, check_global_views=False)
+        project_id = params["project_id"][0]
```

Comment:
I think when we discussed this over slack and I mentioned adding an extra check for this you said to _not worry too much about it_. 

I can add it but is this really a scenario that can happen? When you call this from the profiling dashboard aren't you actually going to specify the URL param `project=id`? 

I guess you're considering the case where, even if you pass `project=id`, the user might still add it in the searchbox, right?

ps.: from my previous experiments an error would be raised already by `get_snuba_params` if that happens so we wouldn't even get to execute the rest of the code. [Here](https://github.com/getsentry/sentry/blob/master/src/sentry/api/bases/organization_events.py#L157-L168) the check that `get_snuba_params` does.

The only problem that I see with the `get_snuba_params` check is that if `organizations:global-views` feature is enabled that it wouldn't raise any error.

Reply:
I mentioned not to worry too much about it at the time because I didn't realize that the vroom end point required a single project id. Because it does, it now becomes a possibility that it can occur and we need to guard against it